docs/api.rst

```diff
@@ -11505,3 +11505,216 @@ Handling user notes
+
+   .. http:example:: curl wget httpie python-requests
+
+      DELETE /api/1/assets/custom HTTP/1.1
+      Host: localhost:5042
+      Content-Type: application/json;charset=UTF-8
+
+      {"identifier": "aca42c5d-36f1-4a8b-af28-5aeb322748b5"}
+
+   :reqjson str identifier: The identifier of the custom asset you're trying to delete.
+
+   **Example Response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Content-Type: application/zip
+
+      {"result": true, "message": ""}
+
+   :statuscode 200: Custom asset was deleted successfully.
+   :statuscode 400: Provided JSON is in some way malformed.
+   :statuscode 409: No user is currently logged in. No custom asset found. Check error message.
+   :statuscode 500: Internal rotki error.
+
+
+.. http:patch:: /api/(version)/assets/custom
+
+   Doing a PATCH on this endpoint will update the content of an already existing custom asset.
+
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      PATCH /api/1/assets/custom HTTP/1.1
+      Host: localhost:5042
+      Content-Type: application/json;charset=UTF-8
+
+        {
+            "identifier": "de0b49aa-65e1-4b4e-94b5-d1cd3e4baee1",
+            "name": "Hotel",
+            "notes": "Hotel from ma",
+            "custom_asset_type": "real estate"
+        }
+
+   :reqjson str identifier: The unique identifier of the custom asset to update.
+   :reqjson str name: The name of the custom asset.
+   :reqjson str[optional] notes: The notes that serve as a description of the custom asset. This is optional.
+   :reqjson str custom_asset_type: The type/category of the custom asset.
+   :reqjson int[optional] started: The creation timestamp of the custom asset. This is optional.
+   :reqjson str[optional] swapped_for: The asset identifier it was swapped for. This is optional.
+
+   **Example Response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Content-Type: application/zip
+
+      {"result": true, "message": ""}
+
+   :statuscode 200: Custom asset was updated successfully.
+   :statuscode 400: Provided JSON is in some way malformed.
+   :statuscode 409: No user is currently logged in. New custom asset name and type is already being used. Custom asset does not exist.  Check error message.
+   :statuscode 500: Internal rotki error.
+
+
+.. http:put:: /api/(version)/assets/custom
+
+   Doing a PUT on this endpoint will add a new custom asset to the DB.
+
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      PUT /api/1/assets/custom HTTP/1.1
+      Host: localhost:5042
+      Content-Type: application/json;charset=UTF-8
+
+      {
+            "name": "Yatch",
+            "custom_asset_type": "flex"
+      }
+
+   :reqjson str name: The name of the custom asset to be created.
+   :reqjson str[optional] notes: The notes of the custom asset to be created. This is optional.
+   :reqjson str custom_asset_type: The type/category of the custom asset.
```

Comment:
I would place custom_asset_type right after the name since these 2 are the main fields of a custom asset (same for the patch verb)